IMDB-Scraper/scraper.py

```diff
@@ -0,0 +1,65 @@
+import requests
```

Comment:
![image](https://user-images.githubusercontent.com/43691873/110943815-a946fc00-8361-11eb-8348-a77d41784c9b.png)
The output has several issues:
- Printing "Something went wrong" which should not be printed as it breaks the flow of reading.
- Look at Highlighted in yellow: it is printing non extra characters. (\xa0)
- You should not print entries which are not complete as it only floods the terminal. Only display values which are somewhat complete.